test_scripts/API/SendLocation/common_send_location.lua

```diff
@@ -0,0 +1,283 @@
+---------------------------------------------------------------------------------------------------
+-- RC common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.defaultProtocolVersion = 2
+config.ValidateSchema = false
+
+--[[ Required Shared libraries ]]
+local mobile_session = require("mobile_session")
+local json = require("modules/json")
+
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+local commonSteps = require("user_modules/shared_testcases/commonSteps")
+local commonTestCases = require("user_modules/shared_testcases/commonTestCases")
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local hmi_values = require("user_modules/hmi_values")
+
+--[[ Local Variables ]]
+local ptu_table = {}
+local hmiAppIds = {}
+
+local common_send_location = {}
```

Comment:
@Itileda I propose to use module name variable in camelCase:
`local commonSendLocation` 
or
`local commonSL`

Reply:
@aderiabin , fixed in 9ae38a4203ca7367bfbafed1628d5cb1a9e9fe0d